### PR TITLE
Hyplerledger Quilt 0.1.0-SNAPSHOT Apply

### PR DESCRIPTION
**Pull request by KKZ**
# TL;DR

Apply the following libraries.

* crypto-conditions : 0.4.0-SNAPSHOT
* ilp-annotations : 0.1.0-SNAPSHOT
* ilp-core : 0.9.2-SNAPSHOT
* dev-ops : 0.1.0-SNAPSHOT

# Fixes

## Enviroment

* .gitignore : append file.
* application.conf : Change default bind hostname `localhost` from `ledger1`.
* build.gradle
    * Change Kotlin JVM Target `1.8` from `1.6`.
    * Compile target add. (crypto-conditions, ilp-annotations, dev-ops)
    * Modify `launchServer` Task's main Class `BootUpKotlinKt` from `ILPLedgerAPIBootUpVerticle`.
    * Modify `printRandomDSAEd25519PrivPubkey` Task's main class.
* version.prop : 1.0 -> 1.1
* interledger_versining.prop : Add crypto-conditions, ilp-annotations, dev-ops.

## Sources

* Modify `org.interledger.cryptoconditions.PreimageSha256Condition` from `org.interledger.Condition`.
* Modify `org.interledger.cryptoconditions.PreimageSha256Fulfillment` from `org.interledger.Fulfillment`.

### CODE REVIEW

**Comment by earizon** [comment]
@KKZ Thanks for the PR. 

I'm not actively supporting this code right now since an ILP architecture redesign from December 2017 is planned that will move all the logic away from settlement ledgers to connectors and trust-lines/payment-channels between peer connectors. It's still possible to have ledgers supporting  HTLA, but the new "ILPv4" design considers that most of the time settlement ledgers will not be fast enough to use condition/fulfillments. This code can still be used for pedagogic purposes to see core pieces needed for an HTLA  and get used to the main core libraries used by Interledger.

Check https://github.com/interledger/rfcs/issues/359 for full details.

**Comment by KKZ** [comment]
Thanks for reply, I will try to understand the latest specifications.
I only keep my repository for reference.
